scripts/ci-e2e.sh

```diff
@@ -102,6 +102,9 @@ EOF
 
 # initialize a router and cloud NAT
 init_networks() {
+#  gcloud compute shared-vpc enable "$GCP_PROJECT"
```

Comment:
These actions require more permissions. We either need to allow this permission, setup a specific project just for this e2e test, or use the same project (which wont be a shared vpc install). Any ideas @damdo ?

Reply:
Not sure what's best here. @cpanato did this ever come up in the past?

Reply:
if need another project to run together, i would suggest to we talk with sig-testing/sig-infra to see if that is possible when running the prow job